### PR TITLE
Update KIP-81 to match with the latest implementation

### PR DESCRIPTION
**Pull request by blukat29**
- Add link to an example implementation at https://github.com/klaytn/governance-contracts-audit
- Update CnStakingV2 description to mention `gcId`
- Update StakingTracker description to use `gcId`
- Update Voting description about access control and flexible timing
- Clarify that V1 contracts are excluded from voting power calculation

### CODE REVIEW

File: KIPs/kip-81.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ Recognizing a tendency that the member with more KLAY makes a decision that bene
 
 The Klaytn on-chain governance voting will be conducted on smart contracts. Several contracts and accounts interact together in the process. The below diagram shows the relationship between contracts and accounts.
 
+Find an example implementation at [here](https://github.com/klaytn/governance-contracts-audit).
```
Comment (baerae-colin) [inline KIPs/kip-81.md]:
```suggestion
Find an example implementation [here](https://github.com/klaytn/governance-contracts-audit).
```

File: KIPs/kip-81.md [hunk 2]
```diff
@@ -183,26 +196,23 @@ Stored in global mappings, and can be updated at any time. Voter account mapping
...
-        // List of eligible nodes and their staking addresses.
-        // Initialized by createTracker().
-        // Note that even if a GC member's stake falls below MIN_STAKE,
-        // eligibleNodes is not updated accordingly. The eligibleNodes denotes
-        // the number of eligible nodes at the initial recording (i.e. createTracker()).
-        address[] nodeIds; // Representative nodeIds of all GC members
-        mapping(address => address) rewardToNodeId;  // reward address to representative nodeId
-        mapping(address => address) stakingToNodeId; // staking contract to representative nodeId
-        uint256 eligibleNodes; // number of GC members with at least 1 vote
+        // Determined at crateTracker() and does not change.
```
Comment (baerae-colin) [inline KIPs/kip-81.md]:
```suggestion
        // Determined at createTracker() and does not change.
```

File: KIPs/kip-81.md [hunk 3]
```diff
@@ -183,26 +196,23 @@ Stored in global mappings, and can be updated at any time. Voter account mapping
...
+        mapping(uint256 => bool) gcExists;
+        mapping(address => uint256) stakingToGCId;
 
         // Balances and voting powers.
-        // Initialized by createTracker() and updated by refreshStake().
-        mapping(address => uint256) stakingBalances; // staking contract balance
-        mapping(address => uint256) nodeBalances;    // total staking balance of a GC member
-        mapping(address => uint256) nodeVotes;       // voting powers of a GC member
-        uint256 totalVotes;                          // total voting powers of all GC members
+        // First collected at crateTracker() and updated at refreshStake() until trackEnd.
```
Comment (baerae-colin) [inline KIPs/kip-81.md]:
```suggestion
        // First collected at createTracker() and updated at refreshStake() until trackEnd.
```

File: KIPs/kip-81.md [hunk 4]
```diff
@@ -428,50 +284,71 @@ A proposal contains textual descriptions and optionally on-chain transactions. T
...
 
-#### Voting steps
+#### Voting steps overview
 
 Under the default timing settings, a typical governance proposal is handled as follows.
 
 1. When a governance proposal is submitted, it enters a 7-day preparation period where GCs can adjust their voting powers by staking or unstaking KLAYs. The list of voters (i.e., GCs) is finalized at the moment of proposal submission. However, voting powers are finalized at the end of the preparation period.
 2. A 7-day voting period immediately follows after the preparation period.
-3. If there are enough Yes votes, the transactions can be queued within 7 days after voting ends.
+3. If there are enough Yes votes, the transactions can be queued within 14 days after voting ends.
```
Comment (baerae-colin) [inline KIPs/kip-81.md:293]: Don't we need to update the diagram from 7 days to 14 days?

Comment (blukat29) [reply KIPs/kip-81.md:293]: Fixed by [6016371](https://github.com/klaytn/kips/pull/105/commits/6016371c69a5bb45a4de4537f7ce25a4cf58242d).

**Review by baerae-colin — APPROVED** [review]
Not enough time to look at. I approve this now. I will check it later and create a PR if I find anything to revise.

**Pull request merged**